src/posts/paper/13-q-and-a.ts

```diff
@@ -42,9 +42,11 @@ When choosing your budget, you want to factor in  two important aspects: risk to
+We think as many as 1 in 10 COVID cases could potentially lead to long-term negative health consequences for the infected person. Data from the UK's [Office for National Statistics](https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/conditionsanddiseases/bulletins/prevalenceofongoingsymptomsfollowingcoronaviruscovid19infectionintheuk/1april2021) suggests that 13.7% of COVID-infected people may continue to experience symptoms after 12 weeks ("long COVID"). Based on data from the [COVID Symptom Study](https://www.medrxiv.org/content/10.1101/2020.10.19.20214494v1.full.pdf) app, researchers estimated that the proportion of people who experience symptomatic COVID infection and continue to have symptoms after 8 weeks is 5.1%, and 2.2% after 12 weeks.
+
+That would mean that a 1% chance of COVID is the same as up to a 0.1% chance of ongoing negative health consequences from COVID. The average American drives [13,476 miles per year](https://www.fhwa.dot.gov/ohim/onh00/bar8.htm). In 2018 [there were](https://cdan.nhtsa.gov/tsftables/National%20Statistics.pdf) 2,491,000 car-crash-related injuries (of any severity) spread across 3,240,327,000,000 vehicle miles travelled (see the [NHTSA website](https://cdan.nhtsa.gov/tsftables/tsfar.htm) for more detailed breakdowns). [A study of Swedish car crash injuries](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3256772/) found that 11.4% of them met the criteria for at least “1% permanent medical impairment” five years later. (“Permanent medical impairment” is a [legally-relevant concept](https://www.disabilitysecrets.com/resources/understanding-permanent-disability-rating-system.htm) that attempts to quantify disability, and the 1% level was the lowest level of disability that was discussed in the Swedish car crash study.) Multiplying this out, we get a 0.11% chance of ongoing negative health consequences from driving for one year.
```

Comment:
Could you also clean up the language here?

That would mean that a 1% chance of COVID is the same as up to a 0.1% chance of ongoing negative health consequences from COVID. -> ...is the same as a 0.12% chance...